Assets/HoloToolkit/Input/Shaders/SpecularHighlight.shader

```diff
@@ -0,0 +1,177 @@
+﻿//
```

Comment:
I could not find any shader that was easily letting me do highlights. 
@dbastienMS does the new shader you are adding give me something like this?